packages/table-core/src/core/row.ts

```diff
@@ -107,19 +108,24 @@ export const createRow = <TData extends RowData>(
     original,
     depth,
     parentId,
+    _accessorFnsCache: {},
     _valuesCache: {},
     _uniqueValuesCache: {},
     getValue: columnId => {
-      if (row._valuesCache.hasOwnProperty(columnId)) {
+      const column = table.getColumn(columnId)
```

Comment:
The cost of this lookup is probably negligible, however the lookup isn't necessary if the following if-statement returns. Moving it to line 123 avoids this.